definitions/npm/expo-barcode-scanner_v6.x.x/flow_v0.69.0-/expo-barcode-scanner_v6.x.x.js

```diff
@@ -0,0 +1,116 @@
+declare module 'expo-barcode-scanner/@@react-native' {
+  declare export type ViewProps = {};
```

Comment:
`{ ... }` ?

Reply:
```js
func(arg: Object): void;  // Object is any
func(arg: {}): void; // So i use object notation
```
// since 0.104, was added linter rule:
https://github.com/flow-typed/flow-typed/blob/6cb9e99836cbd21153f6de47205e20ca96a5382e/cli/src/commands/runTests.js#L313

And to avoid errors:
```js
func(arg: {}): void;  => func(arg: {...}): void;
```